src/config/channelStates.js

```diff
@@ -5,6 +5,7 @@ export const CHANNEL_CLOSED = "CHANNEL_CLOSED";
 export const CHANNEL_WAITING_FOR_SETTLEMENT = "CHANNEL_WAITING_FOR_SETTLEMENT";
 export const CHANNEL_SETTLED = "CHANNEL_SETTLED";
 export const CHANNEL_WAITING_OPENING = "CHANNEL_WAITING_OPENING";
+export const CHANNEL_UNLOCKED = "CHANNEL_UNLOCKED";
```

Comment:
What happens if a channel is set on `CHANNEL_UNLOCKED` and i want, as an user, to create a channel with the same partner? 

I didnt check the openChannel code but it must support to open a channel again with the same partner if channel is unlocked.

Reply:
https://jirainfuy.atlassian.net/browse/RLP-815 is to fix that

Reply:
It is not appropiate to do so, because the idea is that the developer has the control over it.
Unlocked channels may not be shown if the developer wishes to do so.